event/events.go

```diff
@@ -0,0 +1,198 @@
+package event
+
+import (
+	"errors"
+
+	"github.com/scorum/scorum-go/types"
+)
+
+var wrongEventType = errors.New("wrong type")
```

Comment:
please start error names with the `err` prefix

Reply:
ok